### PR TITLE
Feature budget and categories

### PR DESCRIPTION
**Pull request by imila254**
Income and outcome calculation in main window.

### CODE REVIEW

**Review by barkas90 — APPROVED** [review]

File: Desktop/Main.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ private void Main_Load(object sender, EventArgs e)
         {
             PrepareTransactionListView();
             UpdateTransactionList();
+            IncomeAndOutcomeCalculation(_transactionList);
```
Comment (AleksandrasMaliuginas) [inline Desktop/Main.cs:25]: It's not necessary to pass a transaction list because it is a property of a class.

File: Models/Tip.cs [hunk 2]
```diff
@@ -2,6 +2,10 @@
 {
     public class Tip
     {
+        public Tip()
+        {
+
+        }
```
Comment (AleksandrasMaliuginas) [inline Models/Tip.cs:8]: What is this constructor for?

**Review by AleksandrasMaliuginas — CHANGES_REQUESTED** [review]

File: Models/Tip.cs (hunk 2 above)
Comment (imila254) [reply Models/Tip.cs:8]: Going to remove this.

**Review by AleksandrasMaliuginas — APPROVED** [review]
Perfecto

**Review by AleksandrasMaliuginas — APPROVED** [review]

**Pull request merged**